2_process.R

```diff
@@ -61,5 +62,34 @@ p2_targets_list <- list(
         purrr::map(.,summarize_paired_comids) %>%
         bind_rows()
     }
-  )
+  ),
+  
+
+  ## Melt PRMS_nhdv2_xwalk to get all cols of comids Ids and PRMS idsnh filtered to drb 
+  tar_target(p2_drb_comids, 
+            {p2_prms_nhdv2_xwalk %>%
+               # split df by PRMS_segid() -  
+               split(.,.$PRMS_segid) %>%
+               # loop through each row element of this list and str split such that each new col is a cell
+               lapply(.,function(x){
+                 comids <- data.frame(PRMS_segid = x$PRMS_segid,
+                                      comid=unlist(strsplit(x$comid_all,split=";")))
+               }) %>%
+               bind_rows()
+            }
+  ),
+  ## Filter LC data to the AOI : DRB and join with COMIDs area info and PRMS ids
+  # returns a df with unique comids for aoi + area of comid and NLCD LC percentage attributes
+  tar_target(p2_LC_w_catchment_area,
+             AOI_LC_w_area(area_att = p1_nhd_area_att,
+                       NLCD_LC_df = p1_NLCD_data$NLCD_LandCover_2011,
+                       aoi_comids_df = p2_drb_comids)
+             ),
+  
+  ## Estimate LC proportion in PRMS catchment
+  # returns df with percent LC in PRMS catchment in our AOI
```

Comment:
I know it initially took me a minute to realize that the processed land cover attributes were given in proportions rather than percentages. To save ourselves any confusion down the line, I'd suggest we try to keep the naming consistent across column headers, comments, and targets. We're calculating proportions here, is that right?